stratego/lib/stratego/board/board.ex

```diff
@@ -9,6 +9,7 @@ defmodule Stratego.Board do
         IO.puts "\n"
     end
 
```

Comment:
A small suggestion:

if you had something like:

~~~ elixir
@piece_type_to_human %{
  marshall: "M", general: "G", colonel: "C", ...
}

@piece_types Map.keys(@piece_types_to_human)

def piece_type_to_human(type), do: Map.get(@piece_types_to_human, type, "O")
defp piece_types(), do: @piece_types
~~~

then... (continued)